quattor/types/aquilon.pan

```diff
@@ -77,6 +78,17 @@ type structure_maintenance = {
     "duration"   : long(1..100)
 };
 
+type structure_espinfo = {
+    "description"   ? string with check_esp_chars(SELF)
+    "class"         ? string with match(SELF, '(INFRASTRUCTURE|APPLICATION)')
+    # infrafunction is to give more detail to class for the
```

Comment:
annotation pls. (and ideally for all of them 😉 )

Reply:
I'm not sure I can provide too much of use, most of this came from MS as-is.